Makefile

```diff
@@ -99,6 +99,25 @@ endif
 ifeq ($(GOOS),windows)
 	IS_EXE = ".exe"
 endif
+
+ifeq ($(GOARCH),amd64)
+	IMAGETAG ?= -amd64
+	IMAGETAG_NONE ?=
+else
+	IMAGETAG ?= -$(GOARCH)
+	IMAGETAG_NONE ?= -$(GOARCH)
+endif
+
+.PHONY: makedeploys
+makedeploys:
+	sed "s|\-IMAGETAG_NONE|$(IMAGETAG_NONE)|g" deploy/addons/addon-manager.template > deploy/addons/addon-manager.yaml
```

Comment:
This adds a new dependency for Windows builders ("sed"), but I'm not really sure what the alternative is here.

Reply:
Seems to be windows already has this dependency. I can see other sed commands in Makefile